src/main/scala/dotty/dokka/transformers/ImplicitMembersExtensionTransformer.scala

```diff
@@ -19,23 +19,33 @@ class ImplicitMembersExtensionTransformer(ctx: DokkaContext) extends Documentabl
             val companion = c match 
                 case classlike: DClass => ClasslikeExtension.getFrom(classlike).flatMap(_.companion).map(classlikeMap)
                 case _ => None
-            
-            val implictSources = outerMembers ++ companion.toSeq // We can expand this on companion object from parents, generic etc.
 
-            val MyDri = c.getDri
+            val allParents = c.parents.flatMap { p => 
+                classlikeMap.get(p.dri)
+            }
+
+            val parentCompanions = allParents.flatMap { _ match
+                    case cls: DClasslike => ClasslikeExtension.getFrom(cls).flatMap(_.companion).map(classlikeMap)
+                    case _ => None
+            }     
+
+            val implictSources = outerMembers ++ companion.toSeq ++ parentCompanions // We can expand this on generic etc.
```

Comment:
I think we should create an issue and mentioned it here with `// TODO (#issue_no): We can expand this on generic etc.` style of comment